src/main/java/beam/analysis/plots/PersonVehicleTransitionStats.java

```diff
@@ -32,8 +33,13 @@
     private static Map<String, TreeMap<Integer, Integer>> onRoutes = new HashMap();
     private static Map<String, Integer> modePerson = new HashMap<>();
     private static final String fileName = "tripHistogram";
-    private final int binSize = 300;
-    private final int nofBins = 30 * 3600 / binSize + 1;
+    private  int binSize = 0;
+    private  int nofBins = 0;
+
+    PersonVehicleTransitionStats(BeamConfig beamConfig){
+        binSize = beamConfig.beam().outputs().stats().binSize();
+        nofBins = 30 * 3600 / binSize + 1;
```

Comment:
Consider replacing the magic numbers `30 * 3600` with a meaningful constant. Where those numbers came from? `3600` looks like the number of seconds in one hour and `30` looks like the number of days in a month. It is hard to infer what is the purpose of this combination.